src/sentry/api/endpoints/internal_beacon.py

```diff
@@ -0,0 +1,51 @@
+import logging
+
+from rest_framework import serializers, status
+from rest_framework.response import Response
+
+from sentry.api.base import Endpoint
+from sentry.tasks.beacon import send_beacon_metric
+
+logger = logging.getLogger("beacon")
+
+# These is an arbitrarily picked limit for both the # of batched metrics supported,
+# as well as the size of the dict for each metric
+MAX_LENGTH = 20
```

Comment:
Should this be a `settings` entry and be exposed to the frontend (even in a header in our `400` response would be okay?)

Reply:
The DRF validator will give the reason why it fails in the 400 response

Reply:
It is still not machine-readable though, is it?